esmvaltool/utils/recipe_test_workflow/site/dkrz-singularity-env

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# USAGE dkrz-singularity-env COMMAND
+#
```

Comment:
Please add a comment here to explain why we have a separate environment for running singularity on DKRZ

Reply:
Done in b18df45 :)